api-extractor.json

```diff
@@ -299,7 +299,7 @@
          *
          * DEFAULT VALUE: "warning"
          */
-        "logLevel": "warning"
+        "logLevel": "error"

```

Comment:
I believe we also need to update `extractorMessageReporting` and `tsDocMessageReporting`